Models/Todo.cs

```diff
@@ -0,0 +1,18 @@
+using System.Text.Json.Serialization;
+using TodoNET6.Services;
+
+namespace TodoNET6.Models
+{
+    public class Todo : IEntity<Guid>
+    {
+        [JsonPropertyName("uuid")]
+        public Guid Id { get; set; }
+
+        [JsonPropertyName("todo")]
+        public string? Description { get; set; }
+
+        public DateTime? DueDate { get; set; }
+        public DateTime CreatedDate { get; private set; } = DateTime.UtcNow;
```

Comment:
I was wondering this as well

Reply:
`private set` is used here explicitly because the postgres ORM that I am using in the project uses a property's setter to fill the database column. If you do not explicitly declare a setter then the DB column will not be filled correctly when committing to the database.

`Id` is not set in the same way because sometimes you need to manually set the `Id` field in the ORM system to build queries.